templates/snyk-full.template.yaml

```diff
@@ -3,13 +3,13 @@ AWSTemplateFormatVersion: '2010-09-09'
 Description: Snyk integration on AWS. (qs-1qm9kh0j6)
 Outputs:
   AWSRegion:
-    Description: AWS Region for your Snyk ECR integration.
+    Description: AWS Region for your Snyk integration.
     Value: !Sub '${AWS::Region}'
   SnykECRIntegrationRole:
     Description: Amazon Resource Name (ARN) role for Snyk integration.
     Value: !GetAtt 'ECRIntegrationStack.Outputs.SnykECRIntegrationRole'
   SnykLambdaIntegrationRole:
-    Description: Snyk integration role ARN for AWS Lambda.
+    Description: Snyk integration ARN role for AWS Lambda.
```

Comment:
This should be something like: "ARN of the IAM role used to enable Snyk integration for AWS Lambda resources". Shorten as appropriate.